tests/models/blaise/test_blaise_case_information_model.py

```diff
@@ -1,15 +1,14 @@
 from datetime import datetime
+from typing import Dict
 
 from models.blaise.blaise_case_information_model import BlaiseCaseInformationModel
 
 
-def test_import_case_returns_a_populated_model():
-    questionnaire_name = "LMS2101_AA1"
-
-    case_data_dictionary = {
+def valid_case_data_dictionary() -> Dict:
```

Comment:
Could be a fixture

Reply:
done

Reply:
done in https://github.com/ONSdigital/blaise-totalmobile-services/pull/194